impl/src/test/java/org/jboss/forge/test/roaster/model/UnicodeSourceTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Eclipse Public License version 1.0, available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.jboss.forge.test.roaster.model;
+
+import static org.junit.Assert.assertTrue;
+
+import org.jboss.forge.roaster.Roaster;
+import org.jboss.forge.roaster.model.JavaUnit;
+import org.jboss.forge.roaster.model.source.JavaClassSource;
+import org.jboss.forge.roaster.model.source.JavaSource;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class UnicodeSourceTest
+{
+   @Test
+   public void testUnicodeSource()
+   {
+      JavaUnit source = (JavaUnit) Roaster.parseUnit("public class T日本語<Kカナ>{public<T extends Kカナ> void testにほんご() {}}");

```

Comment:
The code sent by me is basically encoded by UTF-8. 
And the source code default encoding of roaster project is UTF-8
because 
the property '<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>' has already set by including the jboss 19 parent pom.